### PR TITLE
firstGame

### PR DESCRIPTION
**Pull request by Just-Den93**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+let btnGame = document.querySelector('.new-game');
+let currentPlayer = document.getElementById('cur-plr');
+
+let step = false;
+let player = '';
+let score = {
+    'x': 0,
+    'o': 0,
+    'd': 0,
+};
```
Comment (ivan629) [inline script.js:16]: empty line after block

File: script.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+count = 0;
+let playerOne = 'x';
+let playerSecond = '0';
+
+function stepPlayerOne(target) {
+    target.innerHTML = playerOne;
+    target.classList.add('playerOne');
+    count++;
+};
```
Comment (ivan629) [inline script.js:34]: seperate functions with empty lines

File: script.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        [6, 7, 8],
+        [0, 3, 6],
+        [1, 4, 7],
+        [2, 5, 8],
+        [0, 4, 8],
+        [2, 4, 6],
+    ];
+    
+    for(let i = 0; i < comb.length; i++) {
+
```
Comment (ivan629) [inline script.js:85]: remove extra empty line

Comment (ivan629) [reply script.js:85]: move blocks[comb[i][0]].classList.contains('playerOne') &&
            blocks[comb[i][1]].classList.contains('playerOne') &&
            blocks[comb[i][2]].classList.contains('playerOne') to some util

File: script.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        if (blocks[comb[i][0]].classList.contains('playerOne') &&
+            blocks[comb[i][1]].classList.contains('playerOne') &&
+            blocks[comb[i][2]].classList.contains('playerOne')) {
+                setTimeout(() => {
+                    blocks[comb[i][0]].classList.add('active');
+                    blocks[comb[i][1]].classList.add('active');
+                    blocks[comb[i][2]].classList.add('active');
+                    res.innerText = 'Cross Win';
+                }, 100);
```
Comment (ivan629) [inline script.js]: empty line after scope

File: script.js [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+                game.removeEventListener('click', init)
+                score.x += 1
+        }
+
+        else if (blocks[comb[i][0]].classList.contains('playerSecond') &&
```
Comment (ivan629) [inline script.js]: move condition to util

File: script.js [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+        blocks[comb[i][1]].classList.contains('playerSecond') &&
+        blocks[comb[i][2]].classList.contains('playerSecond')) {
+            setTimeout(() => {
+                blocks[comb[i][0]].classList.add('active');
+                blocks[comb[i][1]].classList.add('active');
+                blocks[comb[i][2]].classList.add('active');
+                res.innerText = 'Zero Win';
+            }, 100);
```
Comment (ivan629) [inline script.js]: seperate  functions invokes and settimeout

File: script.js [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+            game.removeEventListener('click', init)
+            score.o += 1
+        }
+
+        else if (count == 9 && i === 7) {
```
Comment (ivan629) [inline script.js:102]: nove condition to util

**Review by iholovach-perfectial — APPROVED** [review]

**Pull request merged**